src/js/page_sections/FormFeedback.js

```diff
@@ -85,10 +88,20 @@ class FormFeedback extends Component {
       });
       return;
     }
-    //TODO: post data as needed
-    // after successful form submit
-    this.setState({
-      stage: 0
+
+    postFeedback({
+      title: this.state.values['site-feedback-options'],
+      description: this.state.values['site-feedback-textarea'],
+      email: this.state.values['site-feedback-email']
+    })
+    .then(({data: data}) => {
```

Comment:
could this line just be `.then(({ data }) => {`

Reply:
yep!